documentation/manual/geometries.md

```diff
@@ -0,0 +1,28 @@
+OSM Feature Geometries
+======================
+
+The [_views_](views.md) provided by the OSHDB API provide direct access to the OSM entities, but also allows to get the [JTS](https://en.wikipedia.org/wiki/JTS_Topology_Suite#Geometry_model) [geometries](https://locationtech.github.io/jts/javadoc/org/locationtech/jts/geom/Geometry.html) of the OSM features corresponding to their state at the requested _snapshot_ or _contribution_ time(s). With this geometry, further operations such as [_filtering_](filters.md#areaOfInterest) or [_aggregation_](aggregation.md#aggregateByGeometry) can be performed. It is also irreplacable during the [_map-reduce_](map-reduce.md#geometry_helpers) step to generate useful results, such as the length of a road network for example.
+
+For some OSM elements, such as nodes, generating geometries is straight forward, for others the conversion requires further knowledge of the [data model](https://wiki.openstreetmap.org/wiki/Elements) and [tagging schema](https://wiki.openstreetmap.org/wiki/Tags) used by OSM. The following document gives an overview of how the OSHDB handles the building of geometries of different OSM object types.
+
+Nodes
+-----
+
+Nodes are always presented as [`Point`](https://locationtech.github.io/jts/javadoc/org/locationtech/jts/geom/Point.html)s. Note that nodes which have never had a tags and are a part of a way or relation are considered to be structural-only points (sometimes called _vertices_), and thus not returned in an OSHDB query when querying all nodes. This is because the OSHDB does not consider them to not be _map features_ by their own. They can however be fetched as members of their parent way or relation objects, if needed.
```

Comment:
a tag

Reply:
on their own

Reply:
> on their own

this one was already correct, wasn't it: _“…depending on their composition…”_?